### PR TITLE
Desktop version of portfolio

### PR DESCRIPTION
**Pull request by zhorabay**
- [ ] This pull request enhances the layout and responsiveness of the website by implementing the following changes:
- Used media queries in a CSS file to make the desktop version look like the template in Figma (Desktop: >=768px).
- Used Grid and Flexbox in order to lay out the elements on the page.
- Ensured adherence to the design provided in the Figma templates, focusing on maintaining consistency with regards to fonts, colors, images, text, and margins.
- These changes improve the overall appearance, layout, and responsiveness of the website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+             </article>
+             <article class="card-work-item">
+                 <article class="individual-card-item">
+                     <h3>Multi-Post Stories <br/> Gain+Glory</h3>
+                     <ul class="languages">
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
```
Comment (usorfaitheloho) [inline index.html:111]: Can you kindly remove this  black border-color you introduced to the button of your project

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+                don't hestiate to contact me.</p>
+                <button type="button" class="wrapper"><a href="#">Get My Resume</a></button>
+            </div>
+            <div class="about-me-skils">
+                <div class="worklanguages">
+                    <div><img src="./images/icon-languages.svg" alt="Image depicting languages"></div>
+                    <div><h3>Languages</h3></div>
+                    <div class="lang-tags">
+                       <ul>
+                          <li>JavaScript</li>
```
Comment (usorfaitheloho) [inline index.html:158]: Can you kindly remove this black border-color you introduced to the button of your project

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @zhorabay  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️

Hi @zhorabay, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

almost

## Highlights 🎉
✅ Professional Readme. 
✅ Linters passing.
✅ Correct use of GitHub Flow.

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] I noticed that **you haven't incorporated the changes requested by the previous reviewer yet.** It would be great if you could address those before submitting your work for another review. Thank you!
   image
   image


------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in @taldr27 your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Review by skyv26 — APPROVED** [review]
## Hi @zhorabay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Nice *UI* design ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 
- Used `gh-pages` for deployment ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**